admin/package.json

```diff
@@ -62,5 +62,10 @@
     "ts-jest": "^29.1.0",
     "ts-node": "~9.1.1",
     "typescript": "^4.8.0"
+  },
+  "overrides": {
+    "@babel/traverse@^7.23.2": "7.23.2",
+    "postcss@<8.4.31": "8.4.31",
+    "tough-cookie@<4.1.3": "4.1.3"
```

Comment:
Hi @DerekRoberts , thank you for taking care of the security update.
FOM does not use these packages directly. I wonder if this `overrides` is recommended for temporary fix for security?
And what if in the future the packages that have these dependencies are upgraded to higher version or maybe switch to use other dependencies?
@MCatherine1994 @basilv

Reply:
I think @babel/traverse and postcss are child dependencies of @angular-devkit/core. Maybe instead of doing a override, we can do a fresh install first.

Reply:
@ianliuwk1019 @MCatherine1994 The parent packages have no updated to handle these security issues.  Ranges have been specified in the overrides to handle these specific subdependencies.  Once the parent packages have updated the overrides become useless and can be deleted.

Reply:
@DerekRoberts thanks for pointing out this override syntax, it makes sense to me for this type of scenario, and as you note is temporary. I'd suggest adding comments explaining what parent package we're waiting to do an update, but of course we cant do comments in json, my biggest complaint with the format.

Reply:
@basilv Not being able to comment is a pain!  We can still make this a bit more clear though, but using this kind of format below.  Is that much better?

```
"overrides": {{parent: { oldVer: newVer }},
```